### PR TITLE
Set up Project and UI for basic website done

### PR DESCRIPTION
**Pull request by snehaM2210**
I have set up the project in react and have also designed the landing page which looks like this...
This webpage is also perfectly responsive. This solves issue no. #3 and #6 .
image

### CODE REVIEW

**Comment by abhishekpatelmc** [comment]
Next time create separate PR for each Issue So that its easy to assign marks

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```
Comment (abhishekpatelmc) [inline src/App.js:1]: Next time write the components on different jsx
its not good practice to write all code on app js

File: src/app.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
```
Comment (abhishekpatelmc) [inline src/app.css:1]: It applies the same for CSS also 
can create another component and write a child component

**Review by abhishekpatelmc — APPROVED** [review]
Make a separate component for each feature

**Comment by snehaM2210** [comment]
> Next time create separate PR for each Issue So that its easy to assign marks

Sure sir..I will do it from next time onwards.😊

**Comment by snehaM2210** [comment]
> Make a separate component for each feature

Okay..I will be doing it..

**Pull request merged**